pypowerwall/tedapi/vitals_dictionary.py

```diff
@@ -0,0 +1,517 @@
+import json
+from pypowerwall import __version__
```

Comment:
Another general comment about this file: Overuse of the verb "get" -- Not all of these are actual "getters" (as in: They do no nor could we imagine a useful setter compliment).

Think about what the functions are doing. Are they `calculating`, `computing`, `deriving`, `aggregating`, `processing`, etc? Good function names usually have a "verb_dataStructure" theme, and `get` is vastly overused/abused!

Reply:
I've never found myself confused by a get without a corresponding set (I only get annoyed when there's an unintended consequence that isn't indicated by the verb), but I don't care enough to die on that hill so I've just stripped "get" out of them entirely

Reply:
It was something one of my mentors from Google taught me years ago. I've stuck to it since. I concede it is opinion.